storage/clickhouse/clickhouse.go

```diff
@@ -100,6 +102,7 @@ func NewClickHouse(params *ClickHouseParams) (base.Storage, error) {
 		maxTimeSeriesInQuery: params.MaxTimeSeriesInQuery,
 
 		timeSeries: make(map[uint64][]*prompb.Label, 262144),
+		// useClickHouseQuery: true,
```

Comment:
`Matchers.MatchLabels` was also one of the main reasons for high cpu usage and it's still the default behaviour. I was thinking about adding an env variable to toggle this if we want to move the regex burden to ClickHouse. What are your thoughts?

Reply:
that would be good